public/assets/css/404.css

```
@@ -0,0 +1,24 @@
+.status {
+    text-align: center;
+    font-size: 8rem;
+    padding-top: 60px;
+	color: var(--text-shade-1);
+}
+.description {
+    text-align: center;
+    font-size: 1.7rem;
+    margin-top: -10px;
+	color: var(--text-shade-1);
+}
+.shocked-bandwidth {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 70px;
+    margin-bottom: -120px;
+}
+@media screen and (max-width: 900px) {
+    .shocked-bandwidth {
+        margin-bottom: -100px;
+    }
+}
```

You didn't update the spacing of this file. As per my last review, you have a mix of tabs and spaces here and we use tabs

```suggestion
.status {
	text-align: center;
	font-size: 8rem;
	padding-top: 60px;
	color: var(--text-shade-1);
}
.description {
	text-align: center;
	font-size: 1.7rem;
	margin-top: -10px;
	color: var(--text-shade-1);
}
.shocked-bandwidth {
	display: block;
	margin-left: auto;
	margin-right: auto;
	margin-top: 70px;
	margin-bottom: -120px;
}
@media screen and (max-width: 900px) {
	.shocked-bandwidth {
		margin-bottom: -100px;
	}
}
```